pages/k8s/ipv6.md

```diff
@@ -0,0 +1,156 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "IPv6"
+  description: Using IPv6 with Charmed Kubernetes.
+keywords: juju, network, networking
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: ipv6.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+As of Kubernetes 1.19, support for IPv6 is in beta and dual-stack is in alpha.
+Charmed Kubernetes can enable these by including IPv6 CIDRs in the `cidr`
+config for the Calico charm and the `service-cidr` config for the Kubernetes
+Master charm.  These can contain up to two comma-separated values, with the
+first CIDR in the list being the preferred family for pods, or the default for
+services.  You can override the `IPFamily` for a service when creating it.
```

Comment:
I think we should probably drop a link to https://kubernetes.io/docs/concepts/services-networking/dual-stack/ in here somewhere, which gives an idea of the terminology and current status of these features.